### PR TITLE
Update XLA pin 

### PR DESCRIPTION
**Pull request by yeounoh**
Update XLA pin  to `b9ece5154181608ff1acea9b02a1ee7a526af16b` to incorporate auto-sharding patch.

### CODE REVIEW

**Comment by yeounoh** [comment]
Local build succeeded, ResNet number looks good
```
| Training Device=xla:0/3 Epoch=1 Step=600 Loss=0.00287 Rate=1767.51 GlobalRate=612.32 Time=16:12:17
| Training Device=xla:0/1 Epoch=1 Step=600 Loss=0.00287 Rate=1767.52 GlobalRate=604.49 Time=16:12:17
| Training Device=xla:0/2 Epoch=1 Step=600 Loss=0.00287 Rate=1767.34 GlobalRate=620.80 Time=16:12:17
| Training Device=xla:0/0 Epoch=1 Step=620 Loss=0.00274 Rate=1767.76 GlobalRate=619.21 Time=16:12:18
| Training Device=xla:0/2 Epoch=1 Step=620 Loss=0.00274 Rate=1767.80 GlobalRate=634.05 Time=16:12:18
| Training Device=xla:0/3 Epoch=1 Step=620 Loss=0.00274 Rate=1767.57 GlobalRate=625.48 Time=16:12:18
| Training Device=xla:0/1 Epoch=1 Step=620 Loss=0.00274 Rate=1767.57 GlobalRate=617.58 Time=16:12:18
```

**Review by lsy323 — APPROVED** [review]
LGTM, thanks! Pending on CI

**Review by JackCaoG — APPROVED** [review]

**Review by ghpvnist — APPROVED** [review]
LGTM. Thanks for waiting for my changes to land. Could you also update the PR description hash?

**Pull request merged**
